R/01-createDuplicateModal.R

```diff
@@ -0,0 +1,131 @@
+#' Creates a modalDialog for detectDuplicates module
+#'
+#' @param vars dataframe column names
+#' @param session current session
+createDuplicateModal <- function(vars, session) {
+  ns <- session$ns
+  modalDialog(
+    title = "Duplicate Identifier",
+    easyClose = TRUE,
+    size = "l",
+    footer = modalButton("Close (Without Saving Changes)"),
+    tagList(
+      HTML("This section gives you the option to find and export duplicates or remove them from the dataset.
+                You must first select the columns in which you want to search for duplicates.
+                You can then select the degree of similarity, independently for each column, which specifies how similar cases must be to be classified as duplicates.<br><br>
+                For numeric columns, you can choose between 'Exact Match' and 'Rounded Match'.
+                For the 'Exact Match' selection, observations are only classified as duplicated if the values match exactly.
+                For 'Rounded Match', values are classified as identical if they are identical after rounding.
+                For this reason, you can specify a rounding factor if you have selected 'Rounded Match'.
+                A value of 2, for example, means that the values are rounded to 2 decimal places before being checked for equality.
+                A value of 0 means that the values are rounded to whole numbers.
+                You can also enter negative numbers, which will result in rounding to the power of ten, e.g. the value -2 rounds to the nearest hundred.<br>
+                For text columns you can choose between 'Case Sensitive', where a distinction is made between upper and lower case letters and
+                'Case Insensitive', where no distinction is made.<br><br>
```

Comment:
Can you make a note after this sentence, that per default for all selected columns the most strict  method for matching is used, e.g. _no rounding_ and _case sensitive_ ?